static/app/views/profiling/content.tsx

```diff
@@ -179,7 +179,7 @@ function ProfilingContent({location}: ProfilingContentProps) {
                     'Profiling collects detailed information in production about the functions executing in your application and how long they take to run, giving you code-level visibility into your hot paths.'
                   )}
                 />
-                <FeatureBadge type="beta" />
+                {isProfilingGA ? null : <FeatureBadge type="beta" />}
```

Comment:
We should switch this to a `new` badge like the sidebar

Reply:
@Zylphrex how long is that usually kept around for btw?

Reply:
i dont think we have a predefined time we keep that around but ~1 month should be sufficient